aws-samples/amazon-chime-sdk
# Fix Webpack infinite reloading

**Issue #80 :**

**Description of changes:**
* Update the `webpack.config.dev.js` to not use `InlineChunkHtmlPlugin` which cause the infinite reloading issue.
* Remove the logic of sending HTML page from `server.js`.
  * webpack-dev-server now handles the rendering of HTML pages.
  * `server.js` handles Chime API calls only.
* Add some missing trailing commas.
* Add boundary of `webpack-dev-server` proxy.
* Remove `source-map-loader` dependency from `package.json`, because we don't need it anymore.

**Testing**

1. How did you test these changes?
   * Run the demo locally, make some change, and confirm the live reload works well.
   * Deploy the demo and confirm it works well.

2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it?
Yes, meeting demo.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
